BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Does this file need a copyright header?

Reply:
grpc-java does not have, grpc (c++) has. I don't know, any suggestions?

Reply:
I'd suggest to keep the header as before.

Reply:
Done.